app/pings.yaml

```diff
@@ -11,3 +11,13 @@ activation:
     include the client_id, since it might be reporting an hashed version of the
     Google Advertising ID.
   include_client_id: false
+history_sync:
```

Comment:
Same for this file: this can live in the a-c's component code. Please note that this will require data review.